contracts/test/StakeTable.t.sol

```diff
@@ -0,0 +1,225 @@
+// SPDX-License-Identifier: Unlicensed
+
+/* solhint-disable contract-name-camelcase, func-name-mixedcase, one-contract-per-file */
+
+pragma solidity ^0.8.0;
+
+// Libraries
+import "forge-std/Test.sol";
+import { BN254 } from "bn254/BN254.sol";
+import { BLSSig } from "../src/libraries/BLSSig.sol";
+import { EdOnBN254 } from "../src/libraries/EdOnBn254.sol";
+import { IStakeTable } from "../src/interfaces/IStakeTable.sol";
+import { LightClient } from "../src/LightClient.sol";
+
+// Token contract
+import { ExampleToken } from "../src/ExampleToken.sol";
+
+// Target contract
+import { StakeTable as S } from "../src/StakeTable.sol";
+
+contract StakeTable_register_Test is Test {
+    event Registered(bytes32, uint64, IStakeTable.StakeType, uint256);
+
+    S public stakeTable;
+    ExampleToken public token;
+    LightClient public lightClientContract;
+    address public lightClientAddress;
+    address public tokenAddress;
+    address exampleTokenCreator;
```

Comment:
why do we need the 3 address? the first two can be removed since we already have their contract instances. the last one, you can use [`makeAddr()`](https://book.getfoundry.sh/reference/forge-std/make-addr#makeaddr) to make it clear instead of using `msg.sender` everywhere.

Reply:
You are right. Useless fields removed in 2295635.